cypress/integration/show_work_order.spec.js

```diff
@@ -42,20 +42,7 @@ describe('Search for property', () => {
       cy.contains('E9 6PT')
     })
 
-    // Tenure (raisable repair)
-    cy.get('.hackney-property-alerts li.bg-turquoise').within(() => {
-      cy.contains('Tenure: Secure')
-    })
-
-    // Alerts
-    cy.get('.hackney-property-alerts').within(() => {
-      // Location alerts
-      cy.contains('Address Alert: Property Under Disrepair (DIS)')
-
-      // Person alerts
-      cy.contains('Contact Alert: No Lone Visits (CV)')
-      cy.contains('Contact Alert: Verbal Abuse or Threat of (VA)')
-    })
+    cy.checkForTenureAlertDetails()
```

Comment:
I'd prefer this to accept parameters which list out the specific strings expected in the DOM. Otherwise it's an extra mini-mental leap for the test reader to go from the fixture file, here, and then to the function in support to see how it all ties together.

E.g. 

```
cy.checkForTenureAlertDetails(
  'Address Alert: Property Under Disrepair (DIS)', 
  'Contact Alert: No Lone Visits (CV)', 
  'Contact Alert: Verbal Abuse or Threat of (VA)'
)
```